app/helpers/active_admin/chat/chat_helper.rb

```diff
@@ -0,0 +1,13 @@
+module ActiveAdmin
+  module Chat
+    module ChatHelper
+      def admin_class(message)
+        'admin' if message.sender.class == ActiveAdmin::Chat.admin_user_klass
```

Comment:
What do you think of `is_a?` ?
```suggestion
        'admin' if message.sender.is_a?(ActiveAdmin::Chat.admin_user_klass)
```

Reply:
I like it, but let's do that as part of a separate PR